RecoTracker/TkNavigation/plugins/SimpleNavigableLayer.h

```diff
@@ -32,6 +32,13 @@ class dso_hidden SimpleNavigableLayer : public NavigableLayer {
 
   void setCheckCrossingSide(bool docheck) {theCheckCrossingSide = docheck;}
 
+  std::vector<const DetLayer*> 
+  compatibleLayers( NavigationDirection direction) const override = 0;
+
+  std::vector<const DetLayer*> 
+  compatibleLayers( const FreeTrajectoryState& fts, 
+		    PropagationDirection timeDirection) const override {int counter =0; return compatibleLayers(fts,timeDirection,counter);};
```

Comment:
shouldn't this instead be ```return NavigableLayer::compatibleLayers(fts, timeDirection); ``` ?

Also, why was this a bad choice ```using NavigableLayer::compatibleLayers;``` ?

Reply:
The classes derived from SimpleNavigableLayer did not see the function declaration from its base class NavigableLayer.

Reply:
Disregard previous statement.